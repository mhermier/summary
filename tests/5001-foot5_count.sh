#!/bin/sh

utils/generate-test-input.sh 0 100 0 | ./summary --foot 5 --count --count-skipped --count-matched
