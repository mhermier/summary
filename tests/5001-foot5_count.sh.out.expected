... 95 entries skipped
entry-96
entry-97
entry-98
entry-99
entry-100
5/100 entries matched
